ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -37,6 +37,12 @@ function prepare_next_steps() {
 trap 'prepare_next_steps' EXIT TERM
 trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
 
+if [[ "${CLUSTER_TYPE}" == "aws-arm64" ]]; then
+  # Hack to avoid importing arm64 release image by using an image override
+  OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=registry.ci.openshift.org/ocp-arm64/release-arm64:${BRANCH}
+  export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE
+fi
```

Comment:
From where does the `openshift-install` binary used in this file originate, that this override is necessary?

Reply:
To me it looks like the override is always used, but we need a different one from the one that Prow would usually pass.